pkg/README.md

```diff
@@ -333,6 +333,74 @@ Creates a tar file from a list of inputs.
   </tbody>
 </table>
 
+<a name="pkg_zip"></a>
+## pkg_zip
+
+```python
+pkg_zip(name, extension, strip_prefix, srcs)
+```
+
+Creates a zip file from a list of inputs.
+
+<table class="table table-condensed table-bordered table-params">
+  <colgroup>
+    <col class="col-param" />
+    <col class="param-description" />
+  </colgroup>
+  <thead>
+    <tr>
+      <th colspan="2">Attributes</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>name</code></td>
+      <td>
+        <code>Name, required</code>
+        <p>A unique name for this rule.</p>
+      </td>
+    </tr>
+    <tr>
+      <td><code>extension</code></td>
+      <td>
+        <code>String, default to 'zip'</code>
+        <p>
+            The extension for the resulting zipfile. The output
+            file will be '<i>name</i>.<i>extension</i>'.
+        </p>
+      </td>
+    </tr>
+    <tr>
+      <td><code>strip_prefix</code></td>
```

Comment:
I wasn't sure if we wanted this attr on this rule. We talked about opening a proposal for the more flexible `pkg_directory`/etc. rules in #30 and those would cover this functionality. Is it still useful to have it on the root of the rule for simple cases?

Reply:
Let's try leaving it out for now and adding as requested.

Reply:
I removed it